unity-ggjj/Assets/Scripts/UI/AppearingDialogueController.cs

```diff
@@ -37,13 +37,18 @@ public class AppearingDialogueController : MonoBehaviour, IAppearingDialogueCont
     [Tooltip("Add an AudioClip for the default dialogue chirp here")]
     [SerializeField] private AudioClip _defaultDialogueChirpSfx;
     
+    [Tooltip("Specify how often a chirp should play here")]
```

Comment:
To work around accidental divisions by zero, we could limit the input via the Editor with the [`Range()`-attribute](https://docs.unity3d.com/ScriptReference/RangeAttribute.html):
```suggestion
    [Range(1, 10)]
    [Tooltip("Specify how often a chirp should play here")]
```
(Not sure if 10 is a sensible maximum; just randomly picked a number that made sense at first glance.)